packages/site/src/components/ComponentDocExample.js

```diff
@@ -80,13 +81,14 @@ export default class ComponentDocExample extends Component {
       className,
       description,
       hideSource,
+      id,
       scope,
       source,
       title
     } = this.props;
 
     return (
-      <Root className={className}>
+      <Root className={className} id={id}>
```

Comment:
In addition to making these elements targetable, a common idiom is to include links as well.  This way, one can simply copy the link url and use it elsewhere - e.g. share it, paste it in docs, bookmark it, etc.  See the headers in https://facebook.github.io/react/docs/react-api.html as an example.

Reply:
We opted to keep it simple for this first pass, but I was able to add this feature with about 10 mins work, so in it goes. 😃